vulnerabilities/tests/test_data/apache_kafka/test-advisories.json

```diff
@@ -0,0 +1,200 @@
+[
+    {
+        "aliases": [
+            "CVE-2022-34917"
+        ],
+        "summary": "This CVE identified a flaw where it allows the malicious unauthenticated clients to allocate large amounts of memory on brokers. This can lead to brokers hitting OutOfMemoryException and causing denial of service.",
+        "affected_packages": [
+            {
+                "package": {
+                    "type": "maven",
+                    "namespace": null,
+                    "name": "apache_kafka",
+                    "version": null,
+                    "qualifiers": null,
+                    "subpath": null
+                },
+                "affected_version_range": "vers:maven/>=2.8.0|<=2.8.1|!=2.8.2|>=3.0.0|<=3.0.1|!=3.0.2|>=3.1.0|<=3.1.1|!=3.1.2|>=3.2.0|<=3.2.1|!=3.2.3",
+                "fixed_version": null
+            }
+        ],
+        "references": [
+            {
+                "reference_id": "",
+                "url": "https://kafka.apache.org/cve-list",
+                "severities": []
+            },
+            {
+                "reference_id": "CVE-2022-34917",
+                "url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-34917",
+                "severities": []
+            }
+        ],
+        "date_published": null
+    },
+    {
+        "aliases": [
+            "CVE-2021-38153"
+        ],
+        "summary": "Some components in Apache Kafka use Arrays.equals to validate a password or key, which is vulnerable to timing attacks that make brute force attacks for such credentials more likely to be successful. Users should upgrade to 2.8.1 or higher, or 3.0.0 or higher where this vulnerability has been fixed.",
+        "affected_packages": [
+            {
+                "package": {
+                    "type": "maven",
+                    "namespace": null,
+                    "name": "apache_kafka",
+                    "version": null,
+                    "qualifiers": null,
+                    "subpath": null
+                },
+                "affected_version_range": "vers:maven/2.0.0|2.0.1|2.1.0|2.1.1|2.2.0|2.2.1|2.2.2|2.3.0|2.3.1|2.4.0|2.4.1|2.5.0|2.5.1|2.6.0|2.6.1|2.6.2|!=2.6.3|2.7.0|2.7.1|!=2.7.2|2.8.0.|!=2.8.1|<3.0.0",
```

Comment:
Are you sure the range is using maven syntax for all their versions? including some older ones as found in http://archive.apache.org/dist/kafka/ ?

Reply:
@pombredanne I don't understand your question, but I think this is superseded by your decision above to use only `apache` PURLs for now and to add an issue for a `maven` improver.

Reply:
Good point!